src/projectscene/view/clipsview/clipslistmodel.cpp

```diff
@@ -301,6 +301,30 @@ bool ClipsListModel::changeClipTitle(const ClipKey& key, const QString& newTitle
     return ok;
 }
 
+QVariant ClipsListModel::getSortedNeighbor(const ClipKey& key, int offset) const
+{
+    QList<ClipListItem*> sortedList = m_clipList;
+
+    std::sort(sortedList.begin(), sortedList.end(), [](ClipListItem* a, ClipListItem* b) {
```

Comment:
I think we can do sort once in the `reload` method.

Reply:
moved sorting to update() function - that's the place where m_clipList (clips that are visible on a screen) get updated